### PR TITLE
SCAL-173256 fixed pageId for new modular home page

### PR DESCRIPTION
**Pull request by ubaid-ahmed-ts**
Fixed pageId for new UX

### CODE REVIEW

File: src/embed/app.ts [hunk 1]
```diff
@@ -239,8 +239,8 @@ export class AppEmbed extends V1Embed {
      * @param pageId The ID of the page to be embedded.
      */
     private getIFrameSrc() {
-        const { pageId, path } = this.viewConfig;
-        const pageRoute = this.formatPath(path) || this.getPageRoute(pageId);
+        const { pageId, path, modularHomeExperience } = this.viewConfig;
```
Comment (Nirmay15) [inline src/embed/app.ts:242]: Add false as default for modularHomeExperience

File: src/embed/app.ts [hunk 2]
```diff
@@ -278,20 +278,20 @@ export class AppEmbed extends V1Embed {
      *
      * @param pageId The identifier for a page in the ThoughtSpot app.
      */
-    private getPageRoute(pageId: Page) {
+    private getPageRoute(pageId: Page, modularHomeExperience: boolean) {
```
Comment (Nirmay15) [inline src/embed/app.ts:281]: add UTs for each case and please verify added paths locally as well.

**Review by Nirmay15 — CHANGES_REQUESTED** [review]

File: src/embed/app.ts (hunk 2 above)
Comment (ubaid-ahmed-ts) [reply src/embed/app.ts]: Done and verified

File: src/embed/app.ts (hunk 1 above)
Comment (ubaid-ahmed-ts) [reply src/embed/app.ts:242]: Added

**Review by Nirmay15 — APPROVED** [review]

**Pull request merged**